examples/client/Locomotion/src/pages/Profile/Code.js

```diff
@@ -55,14 +69,33 @@ const Code = () => {
         />
         {showErrorText && <ErrorText>{i18n.t('login.vertError')}</ErrorText>}
         <ResendContainer>
-          <ResendText>
-            {i18n.t('onboarding.pages.code.resendCodeText')}
-          </ResendText>
-          <ResendButton
-            onPress={() => navigation.navigate('Phone')}
-          >
-            {i18n.t('onboarding.pages.code.resendCodeButton')}
-          </ResendButton>
+          <Line>
+            <ResendText>
+              {i18n.t('onboarding.pages.code.resendCodeText')}
+            </ResendText>
+          </Line>
+          <Line>
+            <ResendButton
+              disabled={timer > 0}
+              onPress={() => {
+                if (timer === 0) {
+                  navigation.navigate('Phone');
+                }
+              }}
+            >
+              {i18n.t('onboarding.pages.code.resendCodeButton', {
+                seconds: timer,
```

Comment:
need the second param for the resendCodebutton? not only resendCodeTextSeconds?

Reply:
Right. fixed